README.md

```diff
@@ -158,9 +158,11 @@ Fill a development database with some example content:
 2. Run:
 
 ```sh
-docker compose exec app python3 manage.py upload_dev upload_conf_dev.json
+docker compose exec app /manage.py loaddevdata devdata/devdata.json
```

Comment:
Turns out there were more problems with this! I opened another PR to address them: https://github.com/animorphcoop/shared-futures-space/pull/27/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L154